test/parallel/test-assert.js

```diff
@@ -406,7 +406,7 @@ assert.doesNotThrow(function() { assert.ifError(); });
 
 assert.throws(() => {
   assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
-}, /Got unwanted exception: user message/,
+}, /Got unwanted exception:\ntest: user message/,
               'a.doesNotThrow ignores user message');
```

Comment:
@jasnell unfortunately I didn't really understand where indenting was off :(  I hope it was fixed with new commit... but probably not.